### ISSUE TITLE
sidebar entity circles dont display well with large numbers inside

### ISSUE THREAD

**Issue by chutko** [tenders-exposed/elvis-ember#245]
if a "sum" is selected as a node size or connection width in the query builder, the numbers inside circles and squares introduced in the new sidebar design do not count with numbers with more than 2 digits. so sums like 167,7 (millions €) do not display correctly and mess up the look.

**Comment by chutko** [comment]
![screencapture-localhost-4200-network-597097002dcb5627eb000008-suppliers-1500551479248](https://user-images.githubusercontent.com/4135015/28416012-878ac43a-6d52-11e7-829c-85f71d674fdf.png)

i would like to fix this, but i could also use help from @nightsh or @ca1yps0. as you can see, there are too many digits inside the circles. 

we need to create a function which will take the money sum and round it into thousands (23 097 € -> 23K) or millions (476 521 € -> 0.5M) ... or even billions. 
simply put, we need to output the money sum as small and simple as possible. it does not have to be precise though

**Comment by chutko** [comment]
hint: https://stackoverflow.com/questions/10599933/convert-long-number-into-abbreviated-string-in-javascript-with-a-special-shortn

**Comment by ca1yps0** [comment]
from what i know a function like this already exists. i will implement it.
